docs/Data Lake.md

```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
+
+The app provides the functionality to explore the DepMap CCLE data lake. We can query the datalake for three different functionalities for different genes in the application. Each functionality has a separate tab in left for easy access. 
+
+*   Gene Essentiality
+
+*   Gene Cluster
+
+*   CCLE Heatmap
+
+####Gene Essentiality 
+
+Gene Essentiality of the gene describes the efficacy vs selectivity map for a gene based on different models. The terms can be defined as: 
+
+*   **Efficacy** : how essential the gene is in the sensitive cell lines (the more negative the efficacy is, the more essential the gene is).
+
+*   **Selectivity** : how selectively essential the gene is between sensitive and resistant cell lines (the more positive the selectivity is, the more selective the gene is).
+
+As shown in the following image, one can search for a gene for it’s essentiality and press Go to check the scores. 
+
+![Select gene](../img/DepMapCCLE/image5.png)
+
+The search returns a Essentiality map in the left with the searched gene highlighted with Red. The right panel describes the efficacy and selectivity scores of the matched gene and the dependency scores in different models in the datalake. 
+
+![Essentiality](../img/DepMapCCLE/image6.png)
+![Matched genes](../img/DepMapCCLE/image7.png)
+![Dep score](../img/DepMapCCLE/image8.png)
+
+####Gene Cluster
+
+In concept, genes that work together as complexes or pathways should show similar dependency scores across cell lines. With this assumption, 2,492 essential genes were clustered based on the similarity of their dependency scores across 423 cell lines—the resulting clusters group essential genes that likely work together.
+
+As shown in the figure below one can query for their gene of interest to see what genes cluster together with it. To do the search, type in the name of the gene, suggestions come up to help you select the gene of your interest. There is an option to change the cluster size to change the parameters for stringency of clustering. Small cluster equate to stringent cutoffs.  
+
+![Cluster gene select](../img/DepMapCCLE/image9.png)
+
+The query returns a set of efficacy vs selectivity plot of the gene, a t-SNE plot for functional similarity and genes clustered with the searched gene in the form of a table, connectivity map and correlation scores. 
+
+![Eff plot](../img/DepMapCCLE/image10.png)
+
+![tSNE](../img/DepMapCCLE/image11.png)
+
+![Clustered genes](../img/DepMapCCLE/image12.png)
+
+####CCLE Heatmap
+
+CCLE heatmap aids a user in visualizing the cancer specific expression of selected genes in different cell lines models along with the dependency scores of different genes. The heatmap requires three inputs, **Cancer Type** which will help to select the cell lines models, **Dependency Genes** for dependency scores of genes of interest and **Genes for Expression** which will display the expression of the genes. The inputs can be passed as shown in the following figure. 
```

Comment:
CCLE*Heatmap(Italics)
Cancer Types, Dependency Genes, and Genes of expression should be in Italics

Reply:
changed